### PR TITLE
[HIN Dilepton] pPb Run DQMOffline for 80X

### PR DESCRIPTION
**Pull request by goni-zz**
It's backport of #16288 
Need to change for 4 files.
1, 2. modifications
DQMOffline/Trigger/interface/HLTMuonMatchAndPlot.h
-make pt cut variable
DQMOffline/Trigger/src/HLTMuonMatchAndPlot.cc
-set default variable to work same with before
3, 4. customizations
DQMOffline/Muon/python/diMuonHistograms_cfi.py
DQMOffline/Trigger/python/HLTMuonOfflineAnalyzer_cff.py
-to use Era what we want the values during pPb run

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

**Comment by goni** [comment]
Hi, @dmitrijus @slava77 @BetterWang I changed PR that is only move customization function to DQMOfflineHeavyIons_cff.py as Slava's comments.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by goni** [comment]
Hi @slava77 ,
Thank you for your comment on 80X alarming a test. 
I hope #16288 (for 81X)  will be tested soon as well.

**Comment by dmitrijus** [comment]
+1

**Comment by goni** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by dmitrijus** [comment]
+1

**Comment by goni** [comment]
Hi @slava77 @davidlange6 @dmitrijus @cmsbuild ,
I updated this PR rebase on #16394 .
Please test it and move to next step as soon as possible.

**Comment by goni** [comment]
@cmsbuild please test

**Comment by goni** [comment]
Hi @slava77 @davidlange6 @dmitrijus @cmsbuild ,
I updated this PR rebase on #16394 .
and it works well using runTheMatrix.py -l 281.0 and 140.4 in my local.
Could you please test it and move to next step as soon as possible.

File: DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py [hunk 1]
```diff
@@ -30,16 +30,6 @@
...
-diMuonHistos.etaBin = cms.int32(70) #dimuonhistograms mass, bin
-diMuonHistos.etaBBin = cms.int32(70)
-diMuonHistos.etaEBin = cms.int32(70)
-diMuonHistos.etaBinLM = cms.int32(12)
-diMuonHistos.etaBBinLM = cms.int32(12)
-diMuonHistos.etaEBinLM = cms.int32(12)
-diMuonHistos.LowMassMin = cms.double(2.0)
-diMuonHistos.LowMassMax = cms.double(14.0)
-diMuonHistos.HighMassMin = cms.double(55.0)
-diMuonHistos.HighMassMax = cms.double(125.0)
```
Comment (slava77) [inline DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py]: is there an 81X PR with cleanup of this file?
Your currently pending PR
still has these modifications
https://github.com/cms-sw/cmssw/pull/16288/files#diff-59b63a7bb9dd6c9e55764b5600006fdbR34

BTW, as I mentioned in the other PR, if these are needed for HeavyIons scenario in general and should override what we have in the pp scenario defaults, then these edits have to stay here.

Comment (slava77) [inline DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py]: the nominal policy is for everything aiming for 80X to first have an equivalent in 81X

**Comment by goni** [comment]
Hi @slava77 ,
I updated all values regarding "diMuonhistos" same with 81X

**Comment by slava77** [comment]
@cmsbuild please test

File: DQMOffline/Muon/python/diMuonHistograms_cfi.py [hunk 2]
```diff
@@ -25,3 +25,15 @@
                               HighMassMin = cms.double(55.0),
                               HighMassMax = cms.double(155.0)
                               )
+
+from Configuration.StandardSequences.Eras import eras
+eras.pA_2016.toModify(diMuonHistos,
+                      etaBin = 350,
+                      etaBBin = 350,
+                      etaEBin = 350,
+
```
Comment (stahlleiton) [inline DQMOffline/Muon/python/diMuonHistograms_cfi.py]: Nevermind, after checking the code more carefully I see that if the etaBinLM is zero, it uses the same binning as the Z mass plot, which is what is wanted in thus case. Then everything is fine and you can ignore my comment

**Comment by goni** [comment]
Hi @slava77 @dmitrijus @davidlange6 @diguida 
Please, test it and move to next step please.

**Comment by goni** [comment]
@slava77 , Could you test it please?

**Comment by dmitrijus** [comment]
+1

**Comment by BetterWang** [comment]
Hi @davidlange6 @dmitrijus , this PR is getting urgent to meet the next patch release.
Can we get it merged as soon asap?
Thanks

**Comment by davidlange6** [comment]
+1

**Pull request merged**